src/contact.ts

```diff
@@ -1,7 +1,8 @@
 /**
+ *   @ignore
```

Comment:
Could we move `@ignore` to the end of the comment, instead of at the beginning of?